src/java.desktop/macosx/native/libawt_lwawt/font/AWTFont.m

```diff
@@ -569,10 +572,14 @@ static OSStatus CreateFSRef(FSRef *myFSRefPtr, NSString *inPath)
         NSLog(@"Font is : %@", (NSString*)fontname);
 #endif
         jstring jFontName = (jstring)NSStringToJavaString(env, fontname);
+        CFRelease(fontname);
         (*env)->CallBooleanMethod(env, arrayListOfString, addMID, jFontName);
         if ((*env)->ExceptionOccurred(env)) {
+            CFRelease(fds);
```

Comment:
but DeleteLocalRef(jFontName) shouldn't be called here too?

Reply:
Unnecessary. The VM releases it when we return right here (next line)
The other delete is needed because we are in a loop and we don't know how many we might need
and it could exceed what the VM provisions for.